### DISCUSSION TITLE
provide easy and friendly release version

### DISCUSSION THREAD

**Discussion by SleepyRoy** [vlang/v#11795] [Ideas]
Release version should be as easy and friendly as possible to new comers, facilitating V's spread among programmers with different programming level and different familiarity with V. V development is for professionals, but release should not.

For example, some languages like Python just give an install package. You download and then click on it, finished!

**What need to be done to release version?**
(1). Create easy solution (install package e.g.).
(2). Remove dependency, especially professional tools as many as possible (git e.g.)

**Comment by elimisteve** [comment]
Is reliance on git a significant problem, ya think?

**Comment by SleepyRoy** [comment]
@elimisteve IMO not for development, but probably yes for release version. Even if git dependency is preserved, automatic detection and installation should be there.
Some users are not professional programmers (e.g. some in financial areas), and they may just want to use V with fewest steps. Make your git ready, add to PATH, etc these words are not friendly to everyone.

**Comment by andrewboldi** [comment]
They will probably start thinking about creating a binary once they release version 1.
